scripts/update_stable.py

```diff
@@ -0,0 +1,49 @@
+import argparse
+import re
+
+from common import execute_command, env
+from updater import UpdaterBase
+
+WORKFLOW_PATH = ".github/workflows/get-rust-versions.yml"
+RUSTC_VERSION_RE = re.compile(r"[0-9][.][0-9]{2}[.][0-9]")
+WORKFLOW_RUSTC_VERSION_RE = re.compile(r'(?<=STABLE: ")([0-9][.][0-9]{2}[.][0-9])')
+
+
+class StableUpdater(UpdaterBase):
+
+    def _update_locally(self):
+        execute_command("rustup", "default", "stable")
```

Comment:
Don't think updater itself should call rustup. Let's better delegate it to `preparations-for-updates` action. Currently, it sets up nightly by default, but you can add input to specify which channel to install